cpp/src/utilities/integer_utils.hpp

```diff
@@ -30,7 +30,7 @@ namespace cudf {
 namespace util {
 
 template <typename S>
-constexpr inline S round_up_safe(S number_to_round, S modulus) {
```

Comment:
Why was `constexpr` removed?

Reply:
It doesn't compile with constexpr, as discussed in #1438 and #2833